### ISSUE TITLE
Creating SystemVar.ini

### ISSUE THREAD

**Issue by jointri** [opentoonz/opentoonz#3858]
Is opentoonz at the point that the "Creating SystemVar.ini" step in the GNU/Linux install steps no longer needed? Does the code discover it itself now?

**Comment by pojienie** [comment]
I... think so? I seriously don't remember if I had done that step at all, not even in 1.4 era.

I just tested it by removing $HOME/.config/Opentoonz and copied `stuff` directory from the codebase to $HOME/.config/Opentoonz. After launching Opentoonz, I saw that SystemVar.ini was automatically generated.

My Linux distro is Artix Linux and codebase is latest master (27bd0ac41a10f5d57a4639c43cddd3bb06c38a54)
